### ISSUE TITLE
Improve solution to "The Training Curve" exercise

### ISSUE THREAD

**Issue by tobyhodges** [carpentries-incubator/deep-learning-intro#278]
(With apologies if there is an issue already open about this.)

In my view, the solution provided for [the _The Training Curve_ exercise](https://carpentries-incubator.github.io/deep-learning-intro/02-keras/index.html#the-training-curve) could be improved. At the moment, the exercise and the text before it talk about the potential value of plotting out the loss curve, as a way to evaluate the quality of the training configuration. However, I think it leaves unanswered the question many learners would have at this point - "what does a poorly-configured training process look like and how can I fix it?" I think it could be improved by adding an example of a loss plot for a poorly-configured training process, and by explaining what "jitter" in the curve might indicate/how that could be addressed, etc. Perhaps you could even set another exercise that requires them to generate a loss curve for a poorly-configured training process? (Or that might take too long to achieve?)

**Comment by dsmits** [comment]
Thanks @tobyhodges, you make a good point.
